js/templates/modals/about/btcticker.html

```diff
@@ -0,0 +1,5 @@
+<div class="pureFlex cent currentBTCPrice clrP clrBr clrSh3 tx4 contentBox padSm">
+  <i class="gemSmall" style="background-image: url('../imgs/btcIcon128.png');"></i>
+  $<%= ob.currentBTCPrice %>
```

Comment:
You'll want to remove that $ and use formatCurrency() (it's in the currency module) to get the amount formatted into the user's local currency. For example, in your view's render you can:

```
    loadTemplate('modals/about/btcticker.html', (t) => {
      this.$el.html(t({
        currentBTCPrice: formatCurrency(this.currentBTCPrice, app.settings.get('localCurrency')),
      }));
    });
```

You could also do the formatting in the template, since formatCurrency is available there as a template helper, but completely up to you which way you want to go.

FWIW, you can change your local currency in Settings > General if you want to play around and see the BTC price formatted in different ways. You could also play around with changing the language in that tab, since that also affects the formatting.